sdk/mediaservices/arm-mediaservices/recordings/node/mediaservices_test/recording_mediaservices_create_test.json

```diff
@@ -63,7 +63,7 @@
           "tier": "Standard"
         },
         "kind": "StorageV2",
-        "id": "/subscriptions/92f95d8f-3c67-4124-91c7-8cf07cdbf241/resourceGroups/myjstest/providers/Microsoft.Storage/storageAccounts/mystorageaccountxxx111",
+        "id": "/subscriptions/azure_subscription_id/resourceGroups/myjstest/providers/Microsoft.Storage/storageAccounts/mystorageaccountxxx111",
```

Comment:
I have tried generating the recording, the subscription_id got replaced properly.